### PR TITLE
add support for course completion events

### PR DESCRIPTION
**Pull request by CinecaElearning**

### CODE REVIEW

File: src/Events/CourseCompleted.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    /**
+     * Reads data for an event.
+     * @param [String => Mixed] $opts
+     * @return [String => Mixed]
+     * @override Event
+     */
+     public function read(array $opts) {
+        $version = str_replace("\n", "", str_replace("\r", "", file_get_contents(__DIR__.'/../../VERSION')));
+        return [
+            'user' => $opts['relateduserid'] < 1 ? null : $this->repo->readUser($opts['relateduserid']),
```
Comment (jlowe64) [inline src/Events/CourseCompleted.php]: Like in the other repo, please use the formats that we provide. There are examples of how the data runs through each of these libraries and what it should look like as it exits. Almost none of the info provided here are in the right format. 

As also stated in the other repo, please make a test for your work... Extend the base class and work off that.

**Comment by ryansmith94** [comment]
@jlowe64 might be time to archive this because of inactivity unless you have plans to continue it.

**Comment by CinecaElearning** [comment]
I suppose this pull request is no longer useful. It was made by a colleague of mine on september and the improvements here have been addressed in the pull request I made in https://github.com/LearningLocker/Moodle-Log-Expander/pull/29

**Comment by jlowe64** [comment]
Thanks @CinecaElearning @ryansmith94!

**Pull request closed without merging**